modtools/create-unit.lua

```diff
@@ -25,7 +25,6 @@ local usage = [====[
 modtools/create-unit
 ====================
 Creates a unit.  Usage::
-
```

Comment:
@ lethosor yeah I didn't realize it pushed a new commit, I thought it was just editing the original commit.